compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -1304,6 +1303,9 @@ class Definitions {
   def isInfix(sym: Symbol)(implicit ctx: Context): Boolean =
     (sym eq Object_eq) || (sym eq Object_ne)
 
+  @tu lazy val assumedSuperTraits =
+    Set(ComparableClass, JavaSerializableClass, ProductClass, SerializableClass)
```

Comment:
`SerializableClass` is identical to `JavaSerializableClass` (it used to be different, but since 2.13 scala.Serializable is an alias of java.io.Serializable)